vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -51,6 +41,19 @@ class HostFunctionProxy {
   ChakraRuntime &m_runtime;
 };
 
+constexpr const char *const g_bootstrapBundleSource =
+    "function $$ChakraRuntimeProxyConstructor$$(target, handler)\n"
+    "{\n"
+    "  return new Proxy(target, handler)\n"
+    "}";
+
+constexpr const char *const g_proxyConstructorBootstrapFuncName = "$$ChakraRuntimeProxyConstructor$$";
```

Comment:
>g_proxyConstructorBootstrapFuncName [](start = 28, length = 35)

nit: we should keep these in their original position.

Reply:
I'll revert this.

---
In reply to: [345462359](https://github.com/microsoft/react-native-windows/pull/3650#discussion_r345462359) [](ancestors = 345462359)